docs/interface/Metadata.md

```diff
@@ -1,50 +1,95 @@
 # Anonymous Variables
 
 ## Metadata
+
 Variables can be tagged with a variety of `MetadataFlag` values. These flags
 primarily allow an application to tell Parthenon to apply certain behaviors to
 each field.
 
-### Field Topology
-The following fields specify the topology of the variable, and are exclusive:
-- `Metadata::None`
-- `Metadata::Cell`
-- `Metadata::Face`
-- `Metadata::Edge`
-- `Metadata::Node`
+### Variable Topology
+
+Topology essentially specifies on which place on the finite volume
+grid the variable is defined. These fields specify what index space
+the variable is allocated over. E.g.  `Metadata::Cell` specifies that
+the variable is allocated over the cell index space in each block.
 
-These fields specify what index space the variable is allocated over. E.g.
-`Metadata::Cell` specifies that the variable is allocated over the cell index
-space in each block.
+The following fields specify the topology of the variable, and are
+exclusive:
+
+- `Metadata::None`: no topology specified. The variable could be
+  anywhere, or location is not a meaningful concept for this variable.
+- `Metadata::Cell`: The variable is defined on cell centers. The
+  variable is likely volume-averaged.
+- `Metadata::Face`: The variable is defined on cell faces. The
+  variable is likely area-averaged.
+- `Metadata::Edge`: The variable is defined on cell edges. The
+  variable is likely length-averaged.
+- `Metadata::Node`: The variable is defined at nodes, i.e.,
+  cell-corners. The variable might be volume-averaged, or defined
+  pointwise.
 
 ### Variable Behaviors
-TODO
-- `Metadata::Advected`
-- `Metadata::Conserved`
-- `Metadata::Intensive`
+
+These flags can be used to tell an application code how to treat a
+varaible in relation to the problem.
+
+- `Metadata::Advected` implies a variable is advected with the flow of
+  another variable, e.g., a velocity.
+- `Metadata::Conserved` implies a variable obeys a conservation law.
+- `Metadata::Intensive` implies that the value of a variable does not
+  scale with volume.
 
 ### Output
-TODO
-- `Metadata::Restart`
-- `Metadata::Graphics`
 
-### Tensors
-TODO
-- `Metadata::Vector`
-- `Metadata::Tensor`
+These flags specify how a variable interacts with I/O. Enable them to
+enable output properties.
+
+- `Metadata::Restart` implies a variable is required in restart files
+- `Metadata::Graphics` implies a varaible should be output for visualization
+
+### Tensor properties and boundaries
+
+For multidimensional variables, these flags specify how to treat the
+individual components at boundaries. For concreteness, we will discuss
+reflecting boundaries. But this may apply more broadly. A variable
+with no flag set is assumed to be a *Scalar*. Scalars obey *dirichlet*
```

Comment:
> dirichlet

Gesundheit! I'm guessing this should be capitalized? Just to clarify that this refers to a specific method. In fact, if there's an online resource that describes "Dirichlet boundary conditions", it would be nice if there was a link to that here.

Reply:
Yes it does and yes it should be capitalized. How about I link to wikipedia?